paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -1773,6 +1777,22 @@ internal class DefaultFlowControllerTest {
         }
     }
 
+    @OptIn(ExperimentalCvcRecollectionApi::class)
+    @Test
+    fun `Does not launch CVC Recollection for deferred setup intent`() {
+        cvcRecollectionTest(
+            stripeIntent = PaymentIntentFixtures.PI_REQUIRES_PAYMENT_METHOD,
+            shouldRecollectCvc = false
+        ) { flowController ->
+            flowController.configureWithIntentConfiguration(
+                intentConfiguration = PaymentSheet.IntentConfiguration(
+                    mode = PaymentSheet.IntentConfiguration.Mode.Setup(),
+                    requireCvcRecollection = true
+                )
+            ) { _, _ -> }
```

Comment:
Do we expect this callback to be called? Should we `throw AssertionError` here?

Reply:
If not, it feels like we should be making some assertions.

Reply:
They do get called, just with the result of the configureWithIntentConfiguration call, which isn't needed for this test